app/models/pawn.rb

```diff
@@ -6,4 +6,39 @@ def image
       '&#9823;'
     end
   end
+
+  def valid_move?(x, y)
+    dx = delta_x(x_position, x)
+    dy = delta_y(y_position, y)
+
+    return false if blocked?(x, y)
+
+    # pawn can't move horizontally
+    return false if dx != 0 && dy.zero?
+
+    # pawn moves vertically 1 space
+    return true if dx.zero? && dy == 1
+    return false if color == 'white' && y_position < y
+    return false if color == 'black' && y_position > y
+    false
+  end
+
+  def capture_move?(x, y)
+    x_diff = (x_position - x).abs
+    y_diff = (y_position - y).abs
+    captured_piece = square_occupied?(x, y)
+    return true if x_diff == 1 && y_diff == 1
+    return false if captured_piece.blank?
+    # return false if captured_piece.color ==
```

Comment:
For this line, you can do access the current attribute `color` just by saying `color`, since you're "inside" the piece class.  So,

```ruby
return false if captured_piece.color == color
```

Also, you'll want to add a final condition to the end of this method, and most likely you want to move the one from line 30 down, similar to `valid_move?` above, so it would be something like:

```ruby
def capture_move?(x, y)
  ...
  return false if captured_piece.color == color
  x_diff == 1 && y_diff == 1
end
```